src/utils/hooks/useOpenshiftClusterVersion/utils/constants.ts

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable @cspell/spellchecker */
```

Comment:
Can probably just add whatever word is being flagged to our word list file instead of disabling this check for the whole file.